reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md

```diff
@@ -225,6 +225,55 @@ In general, before you connect and as you are establishing the connection, the
 policies on the local computer are in effect. When you are using the
 connection, the policies on the remote computer are in effect.
 
+## Basic Authentication Limitations on Linux and macOs
+
+When connecting from a Linux or macOs system to Windows, Basic Authentication
+over HTTP is not supported. Basic Authentication can be used over HTTPS by
+installing a certificate on the target server. The certificate must have a
+CN name that matches the hostname, is not expired or revoked. A self-signed
+certificate may be used for testing purposes.
+
+See [How To: Configure WINRM for HTTPS](https://support.microsoft.com/en-us/help/2019527/how-to-configure-winrm-for-https)
+for additonal details.
+
+The following command, run from an elevated command prompt, will configure the
+HTTPS listener with the installed certificate.
+
+```powershell
+$hostinfo = '@{Hostname="<DNS_NAME>"; CertificateThumbprint="<THUMBPRINT>"}'
+winrm create winrm/config/Listener?Address=*+Transport=HTTPS $hostinfo
+```
+
+On the Linux or macOs side, select Basic for authentication and -UseSSl.
+
+```powershell
+# The specified local user must have administrator rights on the target machine.
```

Comment:
Perhaps be explicit that Basic can't be used with domain accounts

Reply:
Fixed